modern/src/SocketController.js

```diff
@@ -26,11 +29,19 @@ const SocketController = () => {
   const [events, setEvents] = useState([]);
   const [notifications, setNotifications] = useState([]);
 
+  const [timeKeepAlive, setTimeKeepAlive] = useState();
```

Comment:
I would probably rename this to something like `lastUpdate`.

Reply:
I'll do that